audit-triplestore-web/src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="
+       http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+       http://camel.apache.org/schema/spring http://camel.apache.org/schema/spring/camel-spring.xsd">
+
+    <bean id="properties" class="org.apache.camel.component.properties.PropertiesComponent">
+        <property name="location" value="classpath:audit.properties"/>
+    </bean>
+
+    <bean id="activemq" class="org.apache.activemq.camel.component.ActiveMQComponent">
+      <property name="brokerURL" value="tcp://localhost:61616"/>
```

Comment:
@escowles I think the `dynamic` section should be removed -- that is specifically for properties used for integration tests.

Also, I'm thinking it makes most sense to break this into `fcrepo.jms.host` and `fcrepo.jms.port` so that it follows existing conventions. If someone really wants to configure the `application.properties` file, they can still do that (e.g. if you wanted to construct a failover URI, etc, etc)